projects/com.oracle.truffle.llvm.parser/src/com/oracle/truffle/llvm/parser/model/functions/FunctionDeclaration.java

```diff
@@ -38,7 +39,7 @@
 import com.oracle.truffle.llvm.runtime.types.symbols.LLVMIdentifier;
 import com.oracle.truffle.llvm.runtime.types.symbols.ValueSymbol;
 
-public final class FunctionDeclaration implements Constant, ValueSymbol {
+public final class FunctionDeclaration implements Constant, ValueSymbol, AtributeGroupGetter {
```

Comment:
Hm, I think it should also work to move the methods of ```AtributeGroupGetter``` into the ```Call``` and ```Invoke``` Interface. Is probably the cleaner solution, even we have a little bit of duplication then.